react/components/atoms/button/button.js

```diff
@@ -31,7 +31,7 @@ export class Button extends PureComponent {
             gradientAngle: 62,
             gradientLocations: [0.4, 0.84],
             gradientColors: ["#4a6fe9", "#6687f6"],
-            width: "100%",
+            width: undefined,

```

Comment:
this is not the same as not setting this value ?

Reply:
Yes, but that's what we're doing everywhere (see the other default props in this component). We do similarly in Vue when we use null.

Reply:
Yup it is the same!
But since we have the same behaviour in `icon: undefined, iconStrokeWidth: undefined` I thought the same should be applied. Also is good practise to apply a default value to non required props, even if the value is the same as if we didn't put anything.
But of course this is not a law so tell me what you would like me to do.